src/UNIV2LPOracle.sol

```diff
@@ -227,8 +228,37 @@ contract UNIV2LPOracle {
         emit Link(id, orb);
     }
 
-    function pass() public view returns (bool ok) {
-        return block.timestamp >= add(zzz, hop);
+    // For consistency with other oracles.
+    function zzz() external view returns (uint256) {
+        if (zph == 0) return 0;  // backwards compatibility
+        return sub(zph, hop);
+    }
+
+    function pass() external view returns (bool /*ok*/) {
+
+        // The below is equivalent to:
+        //
+        //     return block.timestamp >= zph;
+        //
+        // since "x >= y" is equivalent to "!(x < y)".
+        //
+        // Stomping on memory slot zero is safe since
```

Comment:
What do you mean stomping? Typo for storing?

Reply:
No, "stomping" is the word I intended to use...it's fairly common slang to mean "overwriting, particularly in a shared resource". I'll replace it with "overwriting" since that's what I mean anyway and it should be clearer.